sriov_ib_cni_test.sh

```diff
@@ -142,35 +149,72 @@ function exit_code {
     exit $status
 }
 
-status=0
-echo "Creating pod test-pod-1"
-pod_create 'test-pod-1'
-let status=status+$?
+function test_pods {
+    status=0
 
-if [ "$status" != 0 ]; then
-    echo "Error: error in creating the first pod"
-    exit_code $status
-fi
+    echo "Creating pod test-pod-1"
+    pod_create 'test-pod-1'
+    let status=status+$?
+    
+    if [ "$status" != 0 ]; then
+        echo "Error: error in creating the first pod"
+        return $status
+    fi
+    
+    echo "Creating pod test-pod-2"
+    pod_create 'test-pod-2'
+    let status=status+$?
+    
+    if [ "$status" != 0 ]; then
+        echo "Error: error in creating the second pod"
+        return $status
+    fi
 
-echo "Creating pod test-pod-2"
-pod_create 'test-pod-2'
-let status=status+$?
+    echo ""
+    echo "Pods created successfully! VFs GUIDs after pods creation are:"
+    after_creation_guids=$(get_vfs_guids)
+    echo "$after_creation_guids"
+    
+    test_pods_connectivity 'test-pod-1' 'test-pod-2'
+     
+    let status=status+$?
+    
```

Comment:
can you move the pod deleting logic from test_guids reseted ?

basically have:
1. delete pods
2. print guids after deletion
3. test guid reset

Reply:
will do